README.md

```diff
@@ -12,6 +13,48 @@
 └── <a href="./contracts">contracts</a>: Specular L1 contracts
 </pre>
 
+## Getting Started for Developers
+
+### Development Environment Requirements
+
+To contribute to the Specular monorepo, it will be handy to have the Specular environment and
+toolchain set up. The Specular toolchain depends on:
+
+* go
+* Node with pNPM
+
+These can be installed on macOS with:
+
+```
+> brew install go pnpm
+```
+
+and on DEB-based systems:
+
+```
+> apt-get install go pnpm
+```
+
+### Submodule Dependencies
+
+The Specular repository depends on a few other projects that are connected to the Specular
+repository via git submodules including:
+
+* blockscout
+* go-ethereum
+* forge-std
+
+To collect and set up the required submodules, run:
+
+```
+> git submodule init
+> git submodule update --recursive
+```
+
+### Building the Project Locally
+
+TODO
```

Comment:
not sure if you saw this but currently some documentation for building locally lives [here](https://github.com/SpecularL2/specular/tree/main/clients/geth/specular/README.md)

maybe we can link to it from the main readme to make it more obvious?

Reply:
Thanks @siosw! Yes, I was going to either link it to that README or figure out something else. Running through that README what drove me to work on this, at the very least documenting things for myself, but hopefully of some help to others.